timesketch/frontend-ng/src/views/Sketch.vue

```diff
@@ -322,6 +322,15 @@ limitations under the License.
                 </v-list-item-content>
               </v-list-item>
 
+              <v-list-item v-if="meta.permissions.delete" @click="deleteSketch()">
```

Comment:
I get some errors on the console when rendering this part. Looks like `meta.permissions` can be undefined at some point during loading the page and this will result in the following error:
```
[Vue warn]: Error in render: "TypeError: Cannot read properties of undefined (reading 'delete')"
```

Reply:
fixed